test/unit/gen/certificate.go

```diff
@@ -80,6 +80,26 @@ func SetCertificateEmails(emails ...string) CertificateModifier {
 	}
 }
 
+func SetCertificateGivenName(givenName string) CertificateModifier {
```

Comment:
Why 'GivenName' here instead of 'ExtraName'?

Reply:
I thought it would be handy to show how to set a specific ExtraName field, but yeah, it does feel weird. Perhaps it should be more generic and be 2 methods. One that sets an ExtraName field, and in the test, create an ExtraName field and pass it in there.